octodns/record/chunked.py

```diff
@@ -62,9 +63,11 @@ def validate(cls, data, _type):
     def process(cls, values):
         ret = []
         for v in values:
+            # remove leading/trailing whitespace
+            v = v.strip()
             if v and v[0] == '"':
```

Comment:
Throw an error if `v and v[0] == '"' and v[-1] != '"'` instead of silently stripping something off a malformed value.

Reply:
Makes sense & will look into this. Broadly speaking the error handling/checking happens in validate rather than process so that `lenient=True` can do a best effort conversion if told to do so by the user. This would need to be taken into account if there's a shift to a parser style setup as discussed above, it would need to be as forgiving as possible, and need to be able to (more) strictly check things during validate.